stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -123,7 +122,8 @@ public class MessageListViewModel(
     private val channelState: StateFlow<ChannelState?> = chatClient.watchChannelAsState(
         cid = channelId,
         messageLimit = messageLimit,
-        coroutineScope = viewModelScope
+        coroutineScope = viewModelScope,
+        false
```

Comment:
Hey @leandroBorgesFerreira 👋

Can we use a named param here for the sake of consistency?